lib/vanity/metric/active_record.rb

```diff
@@ -99,7 +101,13 @@ def last_update_at
       def after_create(record)
         return unless @playground.collecting?
         count = @ar_column ? (record.send(@ar_column) || 0) : 1
-        call_hooks record.send(@ar_timestamp), nil, [count] if count > 0 && @ar_scoped.exists?(record.id)
+
+        identity = Vanity.context.vanity_identity rescue nil
```

Comment:
It's (very) possible that the after_create callback is firing outside of a valid Vanity.context, in this case we just want to use `nil` as the identity.

This is also the way that Metric::Base deals with formulating the track_args.